drivers/video/video_stm32_dcmi.c

```diff
@@ -174,6 +281,20 @@ static int stm32_dma_init(const struct device *dev)
 		return -EIO;
 	}
 
+#if defined(CONFIG_SOC_SERIES_STM32U5X)
```

Comment:
I suggest checking if this DMAU5 compat shouldn't be check instead of U5 series.